package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Rails would generate the following:

```json
{
  "name": "datagovuk_find",
  "private": true,
  "dependencies": {
      "accessible-autocomplete": "^1.6.1"
  }
}
```

I'm not sure we need the `license`, `repository` and `version` properties.

Reply:
NPM needs all three, but turns out yarn only needs the license, so I'll remove the other two.